utility.py

```
@@ -78,6 +78,81 @@ class StatusIcon(enum.StrEnum):
+    NOT_FOUND = 0xE67E22
+    RATE_LIMIT = 0xF39C12
+    VALIDATION = 0x9B59B6
+    UNEXPECTED = 0xE74C3C
+    TIMEOUT = 0x95A5A6
+
+
+def error_embed(
+    category: ErrorEmbedCategory,
+    title: str,
+    description: str,
+) -> TanjunEmbed:
```

Flake8 (7.3.0)

[error] undefined name 'TanjunEmbed'

(F821)